### DISCUSSION TITLE
An unhandled error has occurred on the Aspire Dashboard

### DISCUSSION THREAD

**Discussion by SvenVandenbrande** [microsoft/aspire#4499] [Q&A] [answered]
I see this in the dashboard

![image](https://github.com/dotnet/aspire/assets/1930936/c938ce69-9f8b-4869-9855-85dcd4d07e44)

DevTools

![image](https://github.com/dotnet/aspire/assets/1930936/fe6e2d77-b748-490d-8123-8f33aa1e2e8f)

Only thing that I added to the default AppHost is the following

![image](https://github.com/dotnet/aspire/assets/1930936/b5b25147-e659-4571-a7bf-22452cdcec58)

How can I find more info about this error?

**Comment by davidfowl** [comment, marked as answer]
See https://github.com/dotnet/aspire/issues/4247

**Reply by SvenVandenbrande** [reply]
Yeah was searching in issues & discussion on the unhandled description. 
That's why I didn't see it. :)

I will also update the issue..
